jooby-jade/README.md

```diff
@@ -0,0 +1,75 @@
+# jade
```

Comment:
move this file to: ```md/jooby-jade```

the site generator will merge this file in the web site and add it here later.